sphinx_gherkindoc/parsers/pytest_bdd.py

```diff
@@ -148,5 +148,7 @@ def description(self):
         """Return description as a list of lines."""
         description = self._data.description
         if description:
-            return description.split("\n")
+            lines = list(description.split("\n"))
+            lines.append("")
```

Comment:
Just add a little comment about and expected blank line at the end. So we know this is important to be here.